pandas/io/formats/format.py

```diff
@@ -1716,7 +1704,7 @@ def _formatter(x):
             x = Timedelta(x)
         result = x._repr_base(format=format)
         if box:
-            result = "'{res}'".format(res=result)
+            result = f"'{result}'"
```

Comment:
I _think_ this may be equivalent to
```suggestion
            result = repr(result)
```

Reply:
I think no because the extra single quotes are needed

Reply:
Oh that's okay, the default way ```repr``` works is to wrap the string in extra quotes for example:

```python
txt = "foo"

str(txt) # foo
repr(txt) # 'foo'
```

Reply:
TIL

Reply:
@WillAyd this thread may explain the reasons of the change

Reply:
We should use the quotes as done originally; this isn't immediately clear to other readers that we would choose `repr` just to have free quoting and is just confusing.